src/applications/mhv/secure-messaging/tests/e2e/pages/PatientMessageDraftsPage.js

```diff
@@ -181,6 +189,12 @@ class PatientMessageDraftsPage {
     cy.wait('@deletedDraftResponse');
   };
 
+  verifyDeleteConfirmMessage = () => {
+    cy.contains('successfully deleted')
+      .focused()
+      .should('have.text', 'Draft was successfully deleted.');
```

Comment:
I would suggest pulling this from `mhv/secure-messaging/util/constants.js` -> `Alerts.Message.DELETE_DRAFT_SUCCESS`